src/main/kotlin/org/jitsi/nlj/transform/node/ToggleablePcapWriter.kt

```diff
@@ -33,6 +36,10 @@ class ToggleablePcapWriter(
     private val pcapLock = Any()
 
     fun enable() {
+        if (!PcapWriterConfig.Config.pcapEnabled()) {
+            // TODO: is this the right exception type, semantically?
+            throw AccessControlException("PCAP capture is disabled in configuration")
```

Comment:
The name seems appropriate but on the other hand this has a very specific meaning and context where it's thrown. Maybe we can create our own subclass or throw a RuntimeException?